src/executables/util_cs.c

```diff
@@ -3391,18 +3408,16 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
       sleep (interval);
     }
   while (interval > 0 && !is_Sigint_caught);
-
```

Comment:
Why do you remove this kind of empty line? if there is not any reason then please revert these changes for code readability. (not only this but also other unnecessary change)

Reply:
Thank you. I will apply it.